src/core_init_atmosphere/mpas_init_atm_cases.F

```diff
@@ -7810,7 +7810,7 @@ subroutine init_atm_case_lbc(timestamp, block, mesh, nCells, nEdges, nVertLevels
       integer, pointer :: index_qv, index_qc, index_qr, index_qi, index_qs, index_qg,index_ni, index_nc, index_nr, index_ns, index_ng
       integer, pointer :: index_nwfa, index_nifa
 ! Chemistry
-      integer, pointer :: index_smoke_fine, index_dust_fine, index_dust_coarse
+      integer, pointer :: index_smoke_fine => null(), index_dust_fine => null(), index_dust_coarse => null()
```

Comment:
@jordanschnell Thanks for a quick fix. But still curious why other points don't need to be initialized?